Traceback (most recent call last):
  File "/Users/kt_imacpro/.pyenv/versions/3.8.5/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/Users/kt_imacpro/.pyenv/versions/3.8.5/lib/python3.8/site-packages/nbclient/client.py", line 1082, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/Users/kt_imacpro/.pyenv/versions/3.8.5/lib/python3.8/site-packages/nbclient/util.py", line 74, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/Users/kt_imacpro/.pyenv/versions/3.8.5/lib/python3.8/site-packages/nbclient/util.py", line 53, in just_run
    return loop.run_until_complete(coro)
  File "/Users/kt_imacpro/.pyenv/versions/3.8.5/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/Users/kt_imacpro/.pyenv/versions/3.8.5/lib/python3.8/site-packages/nbclient/client.py", line 535, in async_execute
    await self.async_execute_cell(
  File "/Users/kt_imacpro/.pyenv/versions/3.8.5/lib/python3.8/site-packages/nbclient/client.py", line 827, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/Users/kt_imacpro/.pyenv/versions/3.8.5/lib/python3.8/site-packages/nbclient/client.py", line 735, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply['content'])
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
import seaborn as sns
from matplotlib import pyplot as plt

print(df.columns[1])
plt.figure()
sns.stripplot(x = 'target', y = df.columns[2], data = df, jitter = 0.2)
plt.show()
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mValueError[0m                                Traceback (most recent call last)
[0;32m<ipython-input-8-059b35ac91fc>[0m in [0;36m<module>[0;34m[0m
[1;32m      4[0m [0mprint[0m[0;34m([0m[0mdf[0m[0;34m.[0m[0mcolumns[0m[0;34m[[0m[0;36m1[0m[0;34m][0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m      5[0m [0mplt[0m[0;34m.[0m[0mfigure[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 6[0;31m [0msns[0m[0;34m.[0m[0mstripplot[0m[0;34m([0m[0mx[0m [0;34m=[0m [0;34m'target'[0m[0;34m,[0m [0my[0m [0;34m=[0m [0mdf[0m[0;34m.[0m[0mcolumns[0m[0;34m[[0m[0;36m2[0m[0;34m][0m[0;34m,[0m [0mdata[0m [0;34m=[0m [0mdf[0m[0;34m,[0m [0mjitter[0m [0;34m=[0m [0;36m0.2[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m      7[0m [0mplt[0m[0;34m.[0m[0mshow[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/.pyenv/versions/3.8.5/lib/python3.8/site-packages/seaborn/_decorators.py[0m in [0;36minner_f[0;34m(*args, **kwargs)[0m
[1;32m     44[0m             )
[1;32m     45[0m         [0mkwargs[0m[0;34m.[0m[0mupdate[0m[0;34m([0m[0;34m{[0m[0mk[0m[0;34m:[0m [0marg[0m [0;32mfor[0m [0mk[0m[0;34m,[0m [0marg[0m [0;32min[0m [0mzip[0m[0;34m([0m[0msig[0m[0;34m.[0m[0mparameters[0m[0;34m,[0m [0margs[0m[0;34m)[0m[0;34m}[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m---> 46[0;31m         [0;32mreturn[0m [0mf[0m[0;34m([0m[0;34m**[0m[0mkwargs[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m     47[0m     [0;32mreturn[0m [0minner_f[0m[0;34m[0m[0;34m[0m[0m
[1;32m     48[0m [0;34m[0m[0m

[0;32m~/.pyenv/versions/3.8.5/lib/python3.8/site-packages/seaborn/categorical.py[0m in [0;36mstripplot[0;34m(x, y, hue, data, order, hue_order, jitter, dodge, orient, color, palette, size, edgecolor, linewidth, ax, **kwargs)[0m
[1;32m   2792[0m         [0mwarnings[0m[0;34m.[0m[0mwarn[0m[0;34m([0m[0mmsg[0m[0;34m,[0m [0mUserWarning[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m   2793[0m [0;34m[0m[0m
[0;32m-> 2794[0;31m     plotter = _StripPlotter(x, y, hue, data, order, hue_order,
[0m[1;32m   2795[0m                             jitter, dodge, orient, color, palette)
[1;32m   2796[0m     [0;32mif[0m [0max[0m [0;32mis[0m [0;32mNone[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m

[0;32m~/.pyenv/versions/3.8.5/lib/python3.8/site-packages/seaborn/categorical.py[0m in [0;36m__init__[0;34m(self, x, y, hue, data, order, hue_order, jitter, dodge, orient, color, palette)[0m
[1;32m   1097[0m                  jitter, dodge, orient, color, palette):
[1;32m   1098[0m         [0;34m"""Initialize the plotter."""[0m[0;34m[0m[0;34m[0m[0m
[0;32m-> 1099[0;31m         [0mself[0m[0;34m.[0m[0mestablish_variables[0m[0;34m([0m[0mx[0m[0;34m,[0m [0my[0m[0;34m,[0m [0mhue[0m[0;34m,[0m [0mdata[0m[0;34m,[0m [0morient[0m[0;34m,[0m [0morder[0m[0;34m,[0m [0mhue_order[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m   1100[0m         [0mself[0m[0;34m.[0m[0mestablish_colors[0m[0;34m([0m[0mcolor[0m[0;34m,[0m [0mpalette[0m[0;34m,[0m [0;36m1[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m   1101[0m [0;34m[0m[0m

[0;32m~/.pyenv/versions/3.8.5/lib/python3.8/site-packages/seaborn/categorical.py[0m in [0;36mestablish_variables[0;34m(self, x, y, hue, data, orient, order, hue_order, units)[0m
[1;32m    151[0m                 [0;32mif[0m [0misinstance[0m[0;34m([0m[0mvar[0m[0;34m,[0m [0mstr[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m    152[0m                     [0merr[0m [0;34m=[0m [0;34m"Could not interpret input '{}'"[0m[0;34m.[0m[0mformat[0m[0;34m([0m[0mvar[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 153[0;31m                     [0;32mraise[0m [0mValueError[0m[0;34m([0m[0merr[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    154[0m [0;34m[0m[0m
[1;32m    155[0m             [0;31m# Figure out the plotting orientation[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m

[0;31mValueError[0m: Could not interpret input 'target'
ValueError: Could not interpret input 'target'

